src/test/java/picard/fingerprint/FingerprintCheckerTest.java

```diff
@@ -203,7 +204,8 @@ public void testCheckFingerprintsSam(final File samFile1, final File samFile2, f
         final List<CrosscheckMetric> metrics = metricsFileReader.getMetrics();
 
         final Map<Set<String>, Set<String>> collected = metrics.stream()
-                .collect(Collectors.groupingBy(s -> CollectionUtil.makeSet(s.LEFT_GROUP_VALUE, s.RIGHT_GROUP_VALUE), Collectors.mapping(s -> s.LOD_SCORE.toString(), Collectors.toSet())));
+//                we sometimes get -0.0, and so the comparison fails...
```

Comment:
unless you're commenting out code, `//` comments should left align with the current indent line.